pkg/subctl/operator/submarinerop/deployment/ensure.go

```diff
@@ -68,6 +68,10 @@ func Ensure(kubeClient kubernetes.Interface, namespace, image string, debug bool
 							Image:           image,
 							Command:         command,
 							ImagePullPolicy: imagePullPolicy,
+							SecurityContext: &v1.SecurityContext{
+								RunAsNonRoot:             &[]bool{true}[0],
+								AllowPrivilegeEscalation: &[]bool{false}[0],
```

Comment:
Change this to new pointer implementation once https://github.com/submariner-io/submariner-operator/pull/1859 is merged.

```suggestion
								AllowPrivilegeEscalation: &[]bool{false}[0],
```